osquery/config/parsers/prometheus_targets.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <osquery/config.h>
+
+namespace pt = boost::property_tree;
```

Comment:
nit: This came up in a PR that I have up right now, we try to avoid dropping aliases in header files. Try to make use of the full path if you don't mind for ease of reading declaration specs.